### ISSUE TITLE
Calibration

### ISSUE THREAD

**Issue by ZetsuKasu** [InvoxiPlayGames/FestivalInstrumentMapper#47]
More then an issue its a question for everyone who is playing with F.I.M, how do you have your calibration settings? I'm playing and it feels with delay but just in the game, i think is an skill issue with the game but if you know how to configure it and works fine, send it to me please. I'm playing with a GH3 Aerosmith Guitar from Xbox. thanks for everything on this project!

**Comment by joeyballentine** [comment]
I agree, I can't seem to nail down the calibration. No matter what it seems off
